Assets/MRTK/Tests/PlayModeTests/BasePlayModeTests.cs

```diff
@@ -19,16 +20,18 @@ namespace Microsoft.MixedReality.Toolkit.Tests
     /// </summary>
     public abstract class BasePlayModeTests
```

Comment:
Should all of our playmode tests just implement this? It seems like the contents of Setup() and TearDown() are identical, and they're virtual if additional things need to happen

Reply:
I think looooong term yes, I was hoping to keep these changes scoped to making things work.

There is still some weirdness though by making all playmode tests just do this, since in those cases you'll still need a yield return null like:

```c#
base.Setup();
// now do my interesting setup
yield return null;
```

So it still... yeah the base setup doesn't add much value there.

There are a few cases where it saves you from having to do stuff, but because there's this weirdness I opted to postpone that decision for this work (i.e. make 2019 tests work)